pkg/sessions/redis/client.go

```diff
@@ -2,14 +2,18 @@ package redis
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/bsm/redislock"
```

Comment:
Good find! I looked at their source code -- the Lua they used to keep the lock/release in a single transaction looks the same as what I did 3-4 years ago.  We must've read the same design article 🤣